src/www/views/servers_view/index.ts

```diff
@@ -42,6 +42,7 @@ Polymer({
         height: -webkit-calc(100vh - 64px);
         font-size: 14px;
         line-height: 20px;
+        padding-bottom: 50px;
```

Comment:
I just quickly hard-coded some padding on the bottom so users can scroll down enough for the bottom card to be fully in view

Reply:
Add a comment?